### PR TITLE
Units Update

### PR DESCRIPTION
**Pull request by selassie-eye**
Updated unit/army classes

### CODE REVIEW

File: src/game/entities/Army.java [hunk 1]
```diff
@@ -6,14 +6,21 @@
...
+      soldiers = new ArrayList<Unit>();
+      this.uuid = UUID.randomUUID();
+    }
 
     public Army(Unit u)
     {
         this.owner = u.owner;
+        soldiers = new ArrayList<Unit>();
         soldiers.add(u);
+        this.uuid = UUID.randomUUID().toString();
```
Comment (karthshen) [inline src/game/entities/Army.java]: Make this consistant

File: src/game/entities/units/Worker.java [hunk 2]
```diff
@@ -17,6 +17,7 @@ public Worker(Structure base)
     this.upkeep = 0.2f;
```
Comment (karthshen) [inline src/game/entities/units/Worker.java]: Change this to 1

**Review by karthshen — DISMISSED** [review]
Comments

**Comment by dkweng** [comment]
Add Resource attribute to Unit. Get Method for resources will return resource attribute times upkeep float; return int rounded after multipllication.

**Review by dkweng — APPROVED** [review]
GJ

**Review by karthshen — APPROVED** [review]
Jk Gj

**Pull request merged**